charts/kubernetes/templates/playbooks/playbooks.yaml

```diff
@@ -0,0 +1,126 @@
+{{- if .Values.playbooks.podSnapshot }}
+---
+apiVersion: mission-control.flanksource.com/v1
+kind: Playbook
+metadata:
+  name: pod-snapshot
+spec:
+  title: Pod Snapshot
+  category: Kubernetes
+  description: Takes a snapshot of the pod (Logs, yaml, & events)
+  components:
+    - type: Kubernetes::Pod
+  actions:
+    - name: Take snapshot
+      exec:
+        script: |
+          path="/tmp/pod-snapshot/$(.component.name)"
```

Comment:
This isn't safe - if component name is `../..` it will delete the root - Rather use `mktemp -d`